cpp/velox/utils/ConfigExtractor.cc

```diff
@@ -189,6 +189,17 @@ std::shared_ptr<facebook::velox::config::ConfigBase> getHiveConfig(
   }
 #endif
 
+#ifdef ENABLE_ABFS
+  std::string_view kSparkHadoopPrefix = "spark.hadoop.";
+  std::string_view kSparkHadoopAbfsPrefix = "spark.hadoop.fs.azure.";
+  for (const auto& [key, value] : conf->rawConfigs()) {
```

Comment:
Please also update `getNativeBackendConf`
https://github.com/apache/incubator-gluten/blob/5fb0448ae2f5d3347b643fb219aecfeda7322802/shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala#L549

Reply:
@zhli1142015 done! I missed this. Thanks.